src/lib.rs

```diff
@@ -21,6 +22,9 @@ pub use ads1x1x::ChannelSelection as adc_Channel;
 
 use std::fmt::Debug;
 
+// Navigator Defaults
+const NAVIGATOR_PWM_XTAL_CLOCK_FREQ: f32 = 24_576_000.0; // Navigator's default crystal clock of 24.5760 MHz
```

Comment:
If we want the comment to be a document:

```suggestion
const NAVIGATOR_PWM_XTAL_CLOCK_FREQ: f32 = 24_576_000.0; /// Navigator's default crystal clock of 24.5760 MHz
```